assets/models/system/facebook-dinov2-image-embeddings-base/description.md

```diff
@@ -0,0 +1,50 @@
+The Vision Transformer (ViT) is a transformer encoder model (BERT-like) pretrained on a large collection of images in a self-supervised fashion.
```

Comment:
nit: it sounds more like vit model description. It doesn't include keyword `dino`. Could you please check.

Reply:
This _is_ a ViT model; DinoV2 is a method to train transformer models.

Will include "DinoV2" so we eliminate any kind of potential confusion.